### PR TITLE
Added new paginator, new grid buttons and new fonts

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: is_core/contrib/background_export/templates/is_core/generic_views/table.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <li>
         {% if export_types %}
         <div class="dropdown columns-dropdown">
-            <button class="btn btn-secondary dropdown-toggle" data-toggle="dropdown"><i class="fa fa-cloud-download"></i> {% trans 'Export' %}<span class="caret"></span></button>
+            <button class="btn btn-secondary dropdown-toggle" data-toggle="dropdown"><i class="fa fa-file-export"></i> <span class="caret"></span></button>
```
Comment (rubickcz) [inline is_core/contrib/background_export/templates/is_core/generic_views/table.html:12]: Can you pls add tooltip for the icon-only buttons? Adding these attributes should be enough:
```
data-toggle="tooltip" title="Tooltip on bottom"
```

File: is_core/templates/is_core/generic_views/base_table.html [hunk 2]
```diff
@@ -65,16 +65,22 @@
...
                     {% endif %}
                 {% endblock %}
-
                 {% block rest-table-buttons %}{% endblock %}
             </ul>
+            <ul class="inline-list inline-list-right table-buttons">
+                <li><button class="btn btn-secondary reset-filters"><i class="fa fa-ban"></i></button></li>
+                <li><button class="btn btn-secondary reload"><i class="fa fa-sync"></i></button></li>
+                <li><button class="btn btn-secondary reload-count"><b>&#8721;</b></button></li>
+            </ul>
```
Comment (rubickcz) [inline is_core/templates/is_core/generic_views/base_table.html:81]: These buttons also deserve tooltip.

**Review by rubickcz — CHANGES_REQUESTED** [review]
Add tooltips pls.

File: is_core/locale/cs/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -306,38 +300,57 @@ msgstr ""
...
+#: templates/is_core/forms/login_form.html:5 templates/is_core/login.html:4
 msgid "Login"
 msgstr "Login"
 
-#: templates/is_core/generic_views/base_table.html:40
-msgid "Exports"
-msgstr "Export"
+#: templates/is_core/generic_views/base_table.html:67
+msgid "Display/hide columns"
+msgstr "Zobrazit/skýrt sloupce"
```
Comment (Formulka) [inline is_core/locale/cs/LC_MESSAGES/django.po:323]: skýrt -> skrýt

**Review by Formulka — COMMENTED** [review]
just a typo

**Review by rubickcz — APPROVED** [review]

**Pull request merged**